rocm-opencl-runtime/PKGBUILD

```diff
@@ -1,49 +1,48 @@
 # Maintainer: Ranieri Althoff <ranisalt+aur at gmail dot com>
 
-_opencl_icd_loader_repo='https://github.com/KhronosGroup/OpenCL-ICD-Loader'
-_opencl_icd_loader_commit='978b4b3a29a3aebc86ce9315d5c5963e88722d03'
-
 pkgname=rocm-opencl-runtime
-pkgver=3.3.0
-pkgrel=6
+pkgver=3.5.0
+pkgrel=1
 pkgdesc='Radeon Open Compute - OpenCL runtime'
 arch=('x86_64')
 url='https://github.com/RadeonOpenCompute/ROCm-OpenCL-Runtime'
 license=('MIT')
-depends=('hsakmt-roct' 'hsa-rocr' 'opencl-icd-loader' 'comgr')
-makedepends=('mesa' 'cmake' 'git' 'rocm-cmake')
+depends=('hsakmt-roct' 'hsa-rocr' 'rocclr' 'opencl-icd-loader')
+makedepends=('cmake' 'rocm-cmake')
 provides=("$pkgname" 'opencl-driver')
-source=("$url/archive/roc-$pkgver.tar.gz"
-        "$_opencl_icd_loader_repo/archive/$_opencl_icd_loader_commit.tar.gz"
-        'install_vendor_file.patch')
-sha256sums=('ac6999f1a491ab066286c2bd6adf50f08f831286f56e267879f9f7eced22f98e'
-            '0c14bf890bd198ef5a814b5b7ed57b69e890b0c0a1bcfba8fdad996fa1a97fc7'
-            'b83de5ea8ae889664ce2725f90c5db8c1c9e98839d75c7743b355d16435dccee')
+source=("$pkgname-$pkgver.tar.gz::$url/archive/roc-$pkgver.tar.gz")
+        # 'install_vendor_file.patch'
+        # 'remove-clinfo.patch'
```

Comment:
If we don't need the patches/if they don't work anymore lets go ahead and delete the patches and their associated lines of code in the PKGBUILD.

Reply:
Done.

Reply:
Seems like one of the patch files is still there?

Reply:
Ups. I've removed it.